pydarn/plotting/power.py

```diff
@@ -0,0 +1,150 @@
+#  Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+#  Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+from pydarn import SuperDARNRadars, exceptions, RTP
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_pwr0_statistic()
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    @classmethod
+    def plot_pwr0_statistic(cls, records: List[dict], beam_num: int = 0,
+                            compare: bool = True, frequency: float = 11000,
```

Comment:
I think it might be worth changing the default for frequency to 'None' and then the code can just use the first record's frequency. I had to go in and manually read the frequency of my data to make this work otherwise.
```suggestion
                            compare: bool = True, frequency: [float, NoneType] = None,
```